app/src/main/java/com/framgia/wsm/screen/confirmrequestoff/ConfirmRequestOffViewModel.java

```diff
@@ -80,6 +84,74 @@ private void initData(OffRequest offRequest) {
             offNoSalaryTo.setOffToPeriod("");
             mOffRequest.setEndDayNoSalary(offNoSalaryTo);
         }
+        if (mActionType == ActionType.ACTION_EDIT) {
+            if (mOffRequest.getAnnualLeave() == null
```

Comment:
WARNING: [CollapsibleIfStatements] These nested if statements could be combined
https://pmd.github.io/pmd-5.5.1/pmd-java/rules/java/basic.html#CollapsibleIfStatements